arkane/ess/psi4.py

```diff
@@ -72,8 +70,8 @@ def get_number_of_atoms(self):
             while line != '' and n_atoms == 0:
                 # Automatically determine the number of atoms
                 if 'Center              X                  Y                   Z               Mass ' in line and n_atoms == 0:
-                    for i in range(2):
-                        line = f.readline()
+                    line = f.readline()
```

Comment:
Super minor but to address the LGTM, could you change this to be `_ = f.readline()` to help indicate that you're only interested in using the information from the 2nd line that is read in (in line 74)?